CONTRIBUTING.md

```diff
@@ -61,6 +61,39 @@ make integration_test
 
 Note the integration tests require Docker, and will take a while to run through.
 
+The ability to receive Postgres logs over syslog is not covered by
```

Comment:
Nice, thanks for adding the section. I haven't tried it but given that you tried, I think it's fine to simply add it.
I think it'd be nice to add some title with this section (maybe with `####`, like `#### Test receiving Postgres logs over syslog`), to make it clear that it's one chunk and kinda optional thing to do. What do you think?

Reply:
Sure. I thought about that, but the rest of the section is not really subdivided. However, this really is a sort of footnote to the testing docs, so I think you're right. I'll adjust.